client/webserver/api.go

```diff
@@ -362,6 +411,14 @@ func (s *WebServer) apiLogout(w http.ResponseWriter, r *http.Request) {
 		SameSite: http.SameSiteStrictMode,
 	})
 
+	http.SetCookie(w, &http.Cookie{
+		Name:     pwKeyCK,
+		Path:     "/",
+		Value:    "",
+		Expires:  time.Unix(0, 0),
+		SameSite: http.SameSiteStrictMode,
+	})
```

Comment:
This is repeated enough to warrant a helper function, `func clearPassCacheCookie(w http.ResponseWriter)`.